helix-term/src/health.rs

```diff
@@ -66,17 +66,19 @@ pub fn general() -> std::io::Result<()> {
         writeln!(stdout, "Language file: default")?;
     }
     writeln!(stdout, "Log file: {}", log_file.display())?;
-    writeln!(stdout, "Runtime directory: {}", rt_dir.display())?;
-
-    if let Ok(path) = std::fs::read_link(&rt_dir) {
-        let msg = format!("Runtime directory is symlinked to {}", path.display());
-        writeln!(stdout, "{}", msg.yellow())?;
-    }
-    if !rt_dir.exists() {
-        writeln!(stdout, "{}", "Runtime directory does not exist.".red())?;
-    }
-    if rt_dir.read_dir().ok().map(|it| it.count()) == Some(0) {
-        writeln!(stdout, "{}", "Runtime directory is empty.".red())?;
+    writeln!(stdout, "Number of Runtime directories: {}", rt_dirs.len())?;
+    for (i, rt_dir) in rt_dirs.iter().enumerate() {
+        writeln!(stdout, "Runtime directory {}: {}", i + 1, rt_dir.display())?;
+        if let Ok(path) = std::fs::read_link(&rt_dir) {
+            let msg = format!("Runtime directory is symlinked to {}", path.display());
+            writeln!(stdout, "{}", msg.yellow())?;
+        }
+        if !rt_dir.exists() {
+            writeln!(stdout, "{}", "Runtime directory does not exist.".red())?;
+        }
+        if rt_dir.read_dir().ok().map(|it| it.count()) == Some(0) {
+            writeln!(stdout, "{}", "Runtime directory is empty.".red())?;
+        }
```

Comment:
I'd just output this similar to `PATH` or `GOPATH`: a bunch of directories separated by `;`

Reply:
In addition to the runtime paths, the original behaviour conditionally reports for each runtime directory:

1. the destination if a runtime directory is symlinked
2. a red warning if the runtime directory doesn't exist
3. a red warning if the runtime directory is empty

I assume you still want this info displayed? How should we clearly display this using a `;`-separated list?  For 2 and 3, we can just filter on the non-existent / empty directories and join them on `;`. Should we just do the same thing for 1, and leave it up to the user to work out *which* runtime directory is the symlink?

Reply:
I've change the implementation so that a `;`-separated list of runtime directories is presented. For the warnings / errors,  changed them all to warnings (yellow) because now it might be more common for a runtime directory to be missing or empty and it not be a problem. I didn't join these paths on `;` in the warnings, instead I just report a warning per case.

One thing is that windows / linux use `;` / `:` for this separator in `PATH` and `GOPATH`. I'm not sure we want to go to the effort to make this OS aware, so I just went with your suggestion for the semicolon. This seems like it would cause the least confusion given windows using `:` after drive letter in path `C:\\` etc.

Let me know what you think.